.github/workflows/gchr_app.yml

```diff
@@ -0,0 +1,150 @@
+# Workflow that builds, tests and then pushes the docker images to the ghcr.io repository
+name: Build and Publish App Image
+
+# Only run one workflow of the same group at a time.
+# There can be at most one running and one pending job in a concurrency group at any time.
+concurrency:
```

Comment:
Which one of the jobs in this workflow are not able to run concurrently? Is it the push and create_manifest?

If so, I would move this concurrency block to the jobs. You also then don't need the "cancel-in-progress".

Reply:
removed the concurrency

Reply:
Okay going to assume that multiple same jobs in this workflow is okay to run concurrently. I guess we will find out if not.